### DISCUSSION TITLE
What are the risks / disadvantages of running 3 server node pool.

### DISCUSSION THREAD

**Discussion by zgub** [minio/minio#14251] [Q&A] [answered]
The documentation specifies that it is recommended to run at least 4 server nodes with one disk each.
However, it is possible to start a pool with three nodes, each one having two disks. So the pool has 6 disks.
What are the disadvantages or risks of such setup?

**Comment by harshavardhana** [comment, marked as answer]
Risk is you have limited HA, and we consider those as TOY setups.

**Comment by zgub** [comment]
What does this exactly mean? 
Limited HA means that it will work, but I can lose ONLY ONE server nodes while still having write possibility?
OR are there further (functionality/stability) restrictions? 
What does a TOY setup mean from functionality/stability point of view?

**Reply by klauspost** [reply]
> What does a TOY setup mean from functionality/stability point of view?

It is fully functional, but with too little failover for us to recommend using it for production workloads.

**Reply by harshavardhana** [reply]
> What does a TOY setup mean from functionality/stability point of view?

A "TOY" setup means it's a "TOY" that's all, not considered a serious deployment.

**Reply by izznogood** [reply]
I don't know if with the new release this post is still relevant but if it can help :
https://github.com/minio/minio/discussions/11425
